### PR TITLE
Fixed Billboards alignment across all axes.

### LINKED ISSUE #139: Billboard class doesn't function correctly.
BuildBoards on device will rotate around the z-axis instead of staying level. The pivot axes are calculated incorrectly.

I have a fix forthcoming.

### PR DESCRIPTION
**Pull request by jwittner**
Fixes #139

### CODE REVIEW

File: Assets/HoloToolkit/Utilities/Scripts/Billboard.cs [hunk 1]
```diff
@@ -43,33 +43,32 @@ private void Awake()
...
                 default:
-                    // No changes needed.
+                    forward = Camera.main.transform.forward;
+                    up = Camera.main.transform.up;
                     break;
             }
 
-            // Calculate and apply the rotation required to reorient the object and apply the default rotation to the result.
-            gameObject.transform.rotation = Quaternion.LookRotation(-directionToTarget) * DefaultRotation;
+
```
Comment (riverar) [inline Assets/HoloToolkit/Utilities/Scripts/Billboard.cs]: Nit: Extra space

File: Assets/HoloToolkit/Utilities/Scripts/Billboard.cs [hunk 2]
```diff
@@ -43,33 +43,32 @@ private void Awake()
...
-            }
+            Vector3 forward;
+            Vector3 up;
 
             // Adjust for the pivot axis.
             switch (PivotAxis)
             {
                 case PivotAxis.X:
-                    directionToTarget.x = gameObject.transform.position.x;
+                    forward = Vector3.ProjectOnPlane(Camera.main.transform.forward, Vector3.right).normalized;
```
Comment (NeerajW) [inline Assets/HoloToolkit/Utilities/Scripts/Billboard.cs]: Adding comments here explaining the math and 'why' for folks will make this so much better :)

Comment (jwittner) [inline Assets/HoloToolkit/Utilities/Scripts/Billboard.cs]: Will do!

Comment (jwittner) [inline Assets/HoloToolkit/Utilities/Scripts/Billboard.cs]: Added!

**Comment by jwittner** [comment]
I updated the code so that the pivot axes lock the objects axes and not the world axes. This is much more flexible and still supports aligning the objects axes to the global if desired.

**Comment by angelaHillier** [comment]
Nice! I look forward to trying this out in my projects :)

**Pull request merged**